### ISSUE TITLE
Square aspect videos not showing

### ISSUE THREAD

**Issue by jd6-37** [plaidman/retromega-next#28]
The videos for Game Boy, GBC, and Neo Geo Pocket aren't showing but I can hear the sound, so I believe they are playing. For some reason they are not visible though. It seems to be an aspect ratio issue because I tested with two videos, both created/exported exactly the same in Handbrake except for the aspect ratio, and the 4:3 video displayed fine while the square video was not visible (but the sound was audible).

Do you think this could be resolved? Of course, I could go and add black borders to all of my GB/GBC/NGP videos to make them 4:3, but this wouldn't be ideal.

Thanks. Amazing theme, btw!

**Comment by plaidman** [comment]
Does this problem exist in other themes? Have you asked around in the discord? I haven't seen this issue but I'm not equipped anymore to try to figure this out.

**Comment by plaidman** [comment]
For reference I used screenscraper.fr to grab my videos. I'm not sure if they keep aspect ratio or not.

**Comment by jd6-37** [comment]
Thanks for the quick reply. I used screenscraper.fr videos too. Game Boy's Adventure Island is one of the videos I'm having issues with. 534x480 MP4.

Actually, wait a second...this appears to be an issue with my device (Andbernic ARC-D w/ GammaOS). The same behavior is happening when I play the videos in my file browser app (Material Files), so it must not be related to the theme. Sorry. It's so odd because the freshly downloaded video from screenscrapr won't play (even though most others do), but if I open it in Handrake and export at 4:3 it works! However, if I export it using the same handbrake settings at 1:1 ratio or the original ratio (534x480), the video will not work. Really odd. I'll figure it out somehow. Sorry for the false alarm!

EDIT/UPDATE: In case anyone else runs into this issue, I downscaled the video to 400x360 (same aspect ratio as the original Game Boy videos) and they now work. I have no idea why that worked. The device plays videos at much higher resolution. It makes no sense to me, but I thought I would share in case it could be of help.
